index.html

```diff
@@ -41,5 +41,141 @@ <h4>LET'S CONNECT</h4>
         </ul>
      </div> 
   </section> 
+  <section class="project-1">
+    <div class="main-image">
+      <img src="./images/SnapshootPortfolio.svg" alt="mainimage">
+    </div>
+    <div class="main-headline">
+      <img src="./images/ProjectTitle.svg" alt="Tonic">
+    </div>
+    <div class="main-role">
+      <img src="./images/frame2.svg" alt="role">
+    </div>
+    <div class="main-text">
+      <img src="./images/Text-2.svg" alt="text">
+    </div>
+    <div class="main-skill">
+      <ul>
+      <li><img src="./images/html.svg" alt="html"></li>
+      <li><img src="./images/css.svg" alt="css"></li>
+      <li><img src="./images/jS.svg" alt="javaScript"></li> 
+    </ul> 
+    </div>
+    <div class="button-1">
+      <form action="./index.html">
+        <button type="submit">See Project</button>
+      </form>
+    </div>
+  </section>
+  <section class="project-1">
+    <div class="main-image">
+      <img src="./images/Portfolio-2.svg" alt="mainimage">
+    </div>
+    <div class="main-headline">
+      <img src="./images/project-title-2.svg" alt="projectname">
+    </div>
+    <div class="main-role">
+      <img src="./images/frame2.svg" alt="role">
+    </div>
+    <div class="main-text">
+      <img src="./images/Text-2.svg" alt="text">
+    </div>
+    <div class="main-skill">
+      <ul>
+      <li><img src="./images/html.svg" alt="html"></li>
+      <li><img src="./images/css.svg" alt="css"></li>
+      <li><img src="./images/jS.svg" alt="javaScript"></li> 
+    </ul> 
+    </div>
+    <div class="button-1">
+      <form action="./index.html">
```

Comment:
- [ ] You don't have to use `form` tag here too, Kindly work on removing it and keep just the button